src/mscorlib/shared/System/Span.NonGeneric.cs

```diff
@@ -341,6 +341,44 @@ internal static unsafe bool EndsWithOrdinalHelper(ReadOnlySpan<char> span, ReadO
             return OrdinalHelper(span.Slice(span.Length - value.Length), value, value.Length);
         }
 
+        /// <summary>
+        /// Helper method for MemoryExtensions.AsSpan(T[] array, int start).
```

Comment:
nit: This isn't a helper method.

Reply:
All the methods in here are called directly by MemoryExtensions (for FastSpan specific implementations), so they are all "helpers" but they do all the work. I tend to just add a summary similar to what is in the public surface. But you have a point.

Reply:
I wanted to avoid the duplicate XML docs. Maintenance annoyance.